src/graphics/GraphvizWiringDiagrams.jl

```diff
@@ -84,6 +84,7 @@ function to_graphviz(f::WiringDiagram;
     port_size::String="24", junction_size::String="0.05",
     outer_ports::Bool=true, anchor_outer_ports::Bool=true,
     graph_attrs::AbstractDict=Dict(), node_attrs::AbstractDict=Dict(),
+    node_colors::Dict{Int,String}=Dict{Int,String}(),
```

Comment:
Would replace with ``node_colors::AbstractDict=Dict()`` to avoid unnecessarily strong typing.